plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogAccessOperations.java

```diff
@@ -265,7 +266,8 @@ public void testRefreshMaterializedView()
 
             assertGlueMetastoreApiInvocations("REFRESH MATERIALIZED VIEW test_refresh_mview_view",
                     ImmutableMultiset.builder()
-                            .addCopies(GET_TABLE, 5)
+                            .addCopies(GET_TABLE, 9)
+                            .addCopies(UPDATE_TABLE, 2)
```

Comment:
6 more calls. Why?

Reply:
The test was disabled before, so the value here was not valid

Reply:
I missed that. Can you make enabling tests a separate commit?